packages/@uppy/transloadit/src/Client.js

```diff
@@ -44,9 +44,12 @@ module.exports = class Client {
       body: data
     }).then((response) => response.json()).then((assembly) => {
       if (assembly.error) {
-        const error = new Error(assembly.error)
+        const error = new Error(assembly)
         error.message = assembly.error
```

Comment:
hmm, I think we can just do `new Error(assembly.error)` and that should set `error.message` correctly too. I don't know why it was done this way with a separate `.message` assignment before :sweat_smile: 

Should we do `error.assembly = assembly` so the template editor can access it that way, rather than parsing `error.details`?